e2e/internal/client/cli.go

```diff
@@ -77,6 +77,26 @@ type SvcDeployInput struct {
 	ImageTag string
 }
 
+// TaskRunInput contains the parameters for calling copilot task run
+type TaskRunInput struct {
+	AppName string
+
+	GroupName string
+
+	Image          string
+	Dockerfile     string
+
+	Subnets        []string
+	SecurityGroups []string
```

Comment:
seems like they are defined but not used

Reply:
Yeah... I had them there because the subnet/security-group test and the image test should get implemented in the future. I'm just not entirely sure how that should be done right now. (Related to [the other question](https://github.com/aws/copilot-cli/pull/1233#discussion_r466059991))